bubbles/plugins/backup_db.py

```diff
@@ -0,0 +1,49 @@
+import os
+import shlex
+import subprocess
+from datetime import datetime
+from pathlib import Path
+
+from bubbles.plugins import BaseCommand
+from bubbles.slack import SlackUtils
+
+
+pg_user = os.environ.get('postgres_user') or ''
+pg_password = os.environ.get('postgres_password') or ''
+pg_db = os.environ.get('postgres_db') or ''
+pg_host = os.environ.get('postgres_host') or ''
+
+
+class BackupPostgres(BaseCommand):
+    trigger_words = ['backup']
+    help_text = 'backup - creates and uploads a full backup of our postgres db.'
+
+    def process(self, msg: str, utils: SlackUtils) -> None:
+        if any([not i for i in [pg_user, pg_password, pg_db, pg_host]]):
+            raise ValueError('I am missing some credentials in my configuration. That means I cannot do that database dump you requested! uwu')
```

Comment:
Please no uwu 🤣

Reply:
Awwwww... But bubbles is supposed to be cute! And uwu is cringe-level cute.

Fair.

Reply:
Just looked at the latest pushed version of the code and I re-implemented this check in another way, subsequently removing the uwu anyway. 😂